src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -504,11 +504,16 @@ func (a *AdminAPI) sendAndReceive(
 ) (*http.Response, error) {
 	var r io.Reader
 	if body != nil {
-		bs, err := json.Marshal(body)
-		if err != nil {
-			return nil, fmt.Errorf("unable to encode request body for %s %s: %w", method, url, err) // should not happen
+		// We might be passing io reader already as body, e.g: license file.
```

Comment:
Can the doc on this function be updated with another line,
` // If the body is already an io.Reader, the reader is used directly without marshaling.`?